MSAL/src/MSAL_Internal.h

```diff
@@ -26,8 +26,8 @@
 //------------------------------------------------------------------------------
 
 #define MSAL_VER_HIGH       0
-#define MSAL_VER_LOW        1
-#define MSAL_VER_PATCH      4
```

Comment:
should we remove "dev" at line 40?

Reply:
Will remove dev when I create a release branch. This PR is still pointing to dev.

Reply:
got it :)